src/components/Nav.svelte

```diff
@@ -88,64 +172,102 @@
       padding-left: 10px;
     }
 
-    .action-item a {
+    .nav__action-btn {
       font-size: 12px;
-      padding: 5px 18px;
+      padding: 6px 18px;
       margin-left: 0;
     }
 
-    header {
+    .nav {
       padding: 0;
     }
 
-    .logo-wrapper img {
+    .nav__links-wrapper {
+      height: 50px;
+    }
+
+    .nav__logo-img {
       width: 110px;
       height: auto;
     }
 
-    .logo-wrapper a {
+    .nav__logo-img {
       margin-left: 20px;
     }
 
-    nav li:not(.action-item) {
-      display: none;
+    .nav__mobile-menu {
+      position: fixed;
+      top: 20px;
+      left: 8px;
+      z-index: 2;
+      cursor: pointer;
+      user-select: none;
```

Comment:
Is these two styles needed? I'm getting tap effect for the whole nav menu.

Reply:
nope, good catch. This was from the old mobile styles.